server/catgenome/src/main/java/com/epam/catgenome/entity/bed/BedFile.java

```diff
@@ -27,12 +27,40 @@
 import com.epam.catgenome.entity.BiologicalDataItemFormat;
 import com.epam.catgenome.entity.FeatureFile;
 import com.epam.catgenome.entity.security.AclClass;
+import com.epam.catgenome.manager.bed.parser.NggbBedCodec;
+import com.epam.catgenome.manager.bed.parser.NggbBedFeature;
+import com.epam.catgenome.manager.bed.parser.NggbMultiFormatBedCodec;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import htsjdk.samtools.util.Tuple;
+import htsjdk.tribble.AsciiFeatureCodec;
+import lombok.SneakyThrows;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
 
 /**
  * Represents a Bed File metadata in the system
  */
 public class BedFile extends FeatureFile {
 
+    public static final Map<String, FileExtension> EXTENSION_MAP = readExtensionMap();
+    private static final String BED = "bed";
+
+    @SneakyThrows
+    private static Map<String, FileExtension> readExtensionMap() {
+        List<FileExtension> extensions = new ObjectMapper().readValue(
```

Comment:
It would be nice to allow to use external file for configuration and configure it location from properties